.env.example

```diff
@@ -1,6 +1,17 @@
-# This file contains Content Security Policy (CSP) directives to test Wagtail's compatibility with CSP.
-# If the variables defined here are loaded into the environment, CSP will be enabled.
+DJANGO_SETTINGS_MODULE=bakerydemo.settings.dev
 
+REDIS_URL=redis://redis
+
+# Database connection settings:
+DATABASE_HOST=db
```

Comment:
`.env` is not just used for the Docker setup - if we make this change, it's going to break the virtualenv setup instructions, since we've not told people to set up a Postgres database there (and, of course, we'd rather not make that a requirement). Maybe there should be a separate `.env.docker.example`?

Reply:
Ah good call, my brain is full of docker at the moment and lost site of the other install options. FYI, I'm wrapping up much more extensive `docker compose` "modernization" work over in https://github.com/wagtail/docker-wagtail-develop/pull/71 which I'll backport here once complete.